crates/bevy_render/src/camera/bundle.rs

```diff
@@ -99,6 +100,7 @@ impl OrthographicCameraBundle<Camera3d> {
             camera: Camera {
                 near: orthographic_projection.near,
                 far: orthographic_projection.far,
+                hdr: true,
```

Comment:
What do Godot, Unreal Engine, and Unity do by default?

Reply:
**Unity:**
|                                        | HDR                                         |
|----------------------------------------|---------------------------------------------|
| Built-in Render Pipeline               | Not by default, enable with Camera.allowHDR |
| Universal Render Pipeline (URP)        | Not by default, enable with Camera.allowHDR |
| High Definition Render Pipeline (HDRP) | Enabled by default, cannot disable it       |

**Godot**
There should be this HDR configuration in the "Environment" (the environment can be attached to cameras or the "WorldEnvironment") but it doesn't seem to be in my godot install.
![godot HDR settings](https://docs.godotengine.org/en/3.0/_images/hdr_parameters.png)

Reply:
That suggests that it should be a high-intent configuration option, as in the user actively configures it rather than it being a default. Default to false everywhere in my opinion.